emcc

```diff
@@ -1385,7 +1386,7 @@ try:
 
   if memory_init_file:
     if shared.Settings.USE_TYPED_ARRAYS != 2:
-      if type(memory_init_file) == int: logging.warning('memory init file requires typed arrays mode 2')
```

Comment:
we have actually deprecated non typed arrays mode 2 code, so that should never be hit